src/modules/account/RadixAccount.ts

```diff
@@ -183,15 +220,97 @@ export default class RadixAccount {
     }
 
     private _onAtomReceived = async (atomObservation: RadixAtomObservation) => {
-        this.processingAtomCounter.next(this.processingAtomCounter.getValue() + 1)
+
+
+        if (!this.processingAtomCounter.closed && !this.processingAtomCounter.isStopped) {
+            this.processingAtomCounter.next(this.processingAtomCounter.getValue() + 1)
+        } else {
+            logger.error(`☢️ processingAtomCounter closed or stopped`)
+        }
 
         atomObservation.processedData = {}
         for (const system of this.accountSystems.values()) {
             await system.processAtomUpdate(atomObservation)
         }
 
-        this.processingAtomCounter.next(this.processingAtomCounter.getValue() - 1)
+        if (!this.processingAtomCounter.closed && !this.processingAtomCounter.isStopped) {
+            this.processingAtomCounter.next(this.processingAtomCounter.getValue() - 1)
+        } else {
+            logger.error(`☢️ processingAtomCounter closed or stopped`)
+        }
+
     }
 
-    
+    public async requestRadsForDevelopmentFromFaucetService(): Promise<RadixTransaction> {
+        return this.requestRadsForDevelopmentFromFaucetServiceWithoutBalanceUpdate()
+            .then((txUnique) => {
+                return this.transferSystem.getAllTransactions()
+                    .delay(2_000)
```

Comment:
Wow, didn't even know you could do underscore separators in numeric literals 😆
That's cool 👍